src/corefx/System.Globalization.Native/icushim.h

```diff
@@ -0,0 +1,237 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+// Enable calling ICU functions through shims to enable support for 
+// multiple versions of ICU if the FEATURE_ICU_VERSION_RESILIENT is
```

Comment:
NIT: I think `FEATURE_ICU_VERSION_RESILIENT` is now spelled `FEATURE_FIXED_ICU_VERSION`?  There's another reference to this in a comment at the end of the file.